packages/fcl-wc/src/service.ts

```diff
@@ -249,3 +264,29 @@ function connectWc(
     }
   }
 }
+
+/**
+ * Show a notification for a WalletConnect request.
+ * @param service - The service that is requesting the user's attention.
+ * @param user - The user that is being requested to sign a transaction.
+ * @returns A close function to dismiss the notification.
+ */
+export function showWcRequestNotification({
+  service,
+  user,
+}: {
+  service: Service
+  user: CurrentUser
+}) {
+  const authnService = user?.services?.find((s: any) => s.type === "authn")
+  const walletProvider = authnService?.provider
+
+  return showNotification({
+    title: walletProvider?.name || "Mobile Wallet",
+    message: isMobile()
+      ? "Tap to view request in app"
```

Comment:
Do we need to start thinking about translations for text like this? Probably not for this PR but might be worth a discussion.

Reply:
I would probably make a larger issue, give it thought to do it right, and address Discovery at the same time. Then give the solution to Flow Wallet too. I've had to do this before and its not as easy as it sounds.